src/content/docs/apm/agents/python-agent/configuration/python-agent-configuration.mdx

```diff
@@ -5284,13 +5284,122 @@ Here are assorted other settings available via the agent configuration file.
     </table>
 
     If this setting is enabled, it will capture package and version information on startup of the agent that is displayed in the APM environment tab.
+
     <Callout variant="tip">
         In applications that have a large number of packages, having this setting enabled may cause a CPU spike as it captures all the package and version information. It is recommended in those cases to disable this setting.
     </Callout>
     <Callout variant="caution">
         Disabling this setting will disable the ability to detect vulnerabilities in outdated packages.
     </Callout>
   </Collapser>
+
+<Collapser
+    id="NEW_RELIC_STARTUP_DEBUG"
+    title="NEW_RELIC_STARTUP_DEBUG"
+  >
+    <table>
+      <tbody>
+        <tr>
+          <th>
+            Type
+          </th>
+
+          <td>
+            Boolean
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            Default
+          </th>
+
+          <td>
+            false
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            [Set in](#options)
+          </th>
+
+          <td>
+            Environment variable
+          </td>
+        </tr>
+        <tr>
+          <th>
+            [Environ variable](#environment-variables)
+          </th>
+
+          <td>
+            `NEW_RELIC_STARTUP_DEBUG`
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    If this setting is enabled, the agent will send detailed troubleshooting messages from its startup scripts directly to your console (standard output). This can be helpful for debugging crashes in the `newrelic-admin` startup script, the alternative `bootstrap/sitecustomize.py` startup script, or the startup sequence of the Kubernetes Agents Operator.
```

Comment:
```suggestion
    If this setting is enabled, the agent will send detailed troubleshooting messages from its startup scripts directly to your console (STDOUT). This can be helpful for debugging crashes in the `newrelic-admin` startup script, the alternative `bootstrap/sitecustomize.py` startup script, or the startup sequence of the Kubernetes Agents Operator.
```